packages/editor-ui/src/components/NodeCreator/ItemIterator.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+	<div>
+		<div
+			:is="transitionsEnabled ? 'transition-group' : 'div'"
+			name="accordion"
+			@before-enter="beforeEnter"
+			@enter="enter"
+			@before-leave="beforeLeave"
+			@leave="leave"
+		>
```

Comment:
dynamic component to avoid animation when searching... would use `<component` but that leads to some console errors with `transition-group`.. 
could not spin this into its own component without losing animation 😞